examples/publish_subscribe_example.py

```diff
@@ -16,8 +16,7 @@ def clock(rpc):
     """
 
     while True:
-        rpc.notify('clock', str(datetime.datetime.now()))
-
+        yield from rpc.notify('clock', str(datetime.datetime.now()))
```

Comment:
Could you replace all `yield from` with `await` when you touch this file anyway? :)